beeline-spring-boot-starter/pom.xml

```diff
@@ -7,7 +7,7 @@
     <parent>
         <groupId>io.honeycomb.beeline</groupId>
         <artifactId>beeline-parent</artifactId>
-        <version>1.1.0</version>
+        <version>1.1.1</version>
```

Comment:
Same here - let's keep it as 1.1.0, we'll do a separate PR for the version bump.

Reply:
As discussed in slack, I have not reverted this change.